packages/vega-typings/types/runtime/index.d.ts

```diff
@@ -257,3 +257,7 @@ export * from 'vega-util';
 export * from './dataflow';
 export * from './renderer';
 export * from './scene';
+
+export { codegen as expressionCodegen, parse as expressionParse } from 'vega-expression';
+
+export { selector as parseSelector } from 'vega-event-selector';
```

Comment:
Would be good to keep the naming scheme parallel. So parseSelector/Expression or selector/expressionParse (I prefer the former).

Reply:
I similarly prefer `parseExpression` over `expressionParse`.

Reply:
Btw, we use `runtimeContext` above. Do you also prefer `codegenExpression` (it sounds less natural than `expressionCodegen` as we go from expression AST to code)?

Reply:
I'm fine with `codegenExpression` for parity with `parseExpression`

Reply:
Done